docs/developer-guide/working-with-rules.md

```diff
@@ -307,6 +309,14 @@ There are also some properties you can access:
 
 You should use a `SourceCode` object whenever you need to get more information about the code being linted.
 
+#### Deprecated
```

Comment:
Should we also add the old methods like `getTokenOrCommentBefore` to this section?